app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
![image](https://user-images.githubusercontent.com/103201010/209556762-7746bea9-475f-4fca-b8ce-a4d96f6b8d30.png)
It should also inherit from ABC class